components/timezone/src/time_zone.rs

```diff
@@ -131,8 +135,12 @@ impl CustomTimeZone {
         }
     }
 
-    /// Parse a [`CustomTimeZone`] from a UTF-8 string representing a GMT Offset. See also [`GmtOffset`].
+    /// Parse a [`CustomTimeZone`] from a UTF-8 string representing a GMT Offset
+    /// or an IANA time zone identifier.
+    ///
+    /// To parse from an IXDTF string, use [`CustomZonedDateTime::try_from_str`].
```

Comment:
question: Does `CustomZonedDateTime` need to be specified here between `CustomZonedDateTime<Iso>` and `CustomZonedDateTime<AnyCalendar>`?

Reply:
No but I do need to import `CustomZonedDateTime` for the link to work. Done